src/views/Dashboard.tsx

```diff
@@ -246,6 +241,11 @@ const Dashboard = () => {
   useKeyPress(["Escape"], clearCreating);
   useKeyPress(["i"], () => setCreatingItem(true), null, true);
 
+  const balancerElementClassname = React.useMemo(
+    () => (creatingItem ? "w-[328.575px]" : "w-[107.406px]"),
```

Comment:
hardcoding pixel size seems a lil sussy, does this work if the screen size changes?